File: Class Representation/Aggregation/Typescript/Aggregation.ts
class Developer {
  private name: string;

  constructor(name: string) {
    this.name = name;
  }

  getName(): string {
    return this.name;
  }
}

class Team {
  private name: string;
  private devs: Developer[] = [];

  constructor(name: string, devs: Developer[]) {
    this.name = name;
    this.devs = devs;
  }

  printDevs(): void {
    console.log(`Devs in ${this.name} Team: `);
    for (let i of this.devs) {
      console.log(`${i.getName()} \n`);
    }
  }
}

File: Class Representation/Aggregation/C++/Aggregation.cpp
#include <iostream>
#include <vector>
using namespace std;

class Developer
{
private:
    string name;

public:
    Developer(string name)
    {
        this->name = name;
    }

    string getName()
    {
        return this->name;
    }
};

class Team
{

private:
    string name;
    vector<Developer *> devs;

public:
    Team(string name, vector<Developer *> devs)
    {
        this->name = name;
        this->devs = devs;
    }

    void printDevs()
    {
        cout << "Devs in " << this->name << " Team " << endl;
        for (int i = 0; i < this->devs.size(); i++)
        {
            cout << devs[i]->getName() << endl;
        }
    }
};

int main()
{

    return 0;
}